### PR TITLE
Add class to manage docker on the run

### LINKED ISSUE #4: Run a Docker image
Create an interactor/service/whatever that is able to run Docker image/images from single file `docker-compose.yml`. All the environment is provided if necessary. Perform the operation using OS subprocessing OR find a library for Docker management.

### PR DESCRIPTION
**Pull request by Dimanaux**
Class to run docker images and control them.
Resolves issue #4

### CODE REVIEW

**Comment by Dimanaux** [comment]
Я не удержался и нажал Ctrl+Alt+L во всех файлах

File: src/main/java/com/github/javalab/javaiaas/DockerImage.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.function.Consumer;
+
+class DockerImage {
```
Comment (AinaArd) [inline src/main/java/com/github/javalab/javaiaas/DockerImage.java]: why so?

File: src/main/java/com/github/javalab/javaiaas/controllers/ApplicationsController.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+    @GetMapping
+    public ResponseEntity<?> index(Authentication authentication) {
+        return ResponseEntity.ok(currentUser(authentication).getApplications());
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity<?> show(Authentication authentication,
+                                  @PathVariable Long id) throws NotFoundException {
+        authorize(authentication, id);
```
Comment (AinaArd) [inline src/main/java/com/github/javalab/javaiaas/controllers/ApplicationsController.java:30]: maybe change on @PreAuthorize("isAuthenticated()") ?

Comment (Dimanaux) [reply src/main/java/com/github/javalab/javaiaas/controllers/ApplicationsController.java:30]: But how to check if current user owns the application?

**Review by AinaArd — APPROVED** [review]

**Pull request merged**